samples/snippets/cpp/VS_Snippets_Remoting/NclMailASync/cpp/mailasync.cpp

```diff
@@ -36,7 +36,7 @@ int main(array<String^>^ args)
 {
     if (args->Length > 1)
     {
-        // Command line argument must the the SMTP host.
+        // Command line argument must the SMTP host.
```

Comment:
Removing the "the" still leaves a nonsensical sentence. Perhaps remove the comment entirely?

Reply:
changed to must be since the host is the parameter for the SmtpClient constructor